http/codegen/openapi/openapi_v2_builder.go

```diff
@@ -110,6 +110,35 @@ func NewV2(root *httpdesign.RootExpr) (*V2, error) {
 	return s, nil
 }
 
+// ExtensionsFromExpr generates swagger extensions from the given metadata
+// expression.
+func ExtensionsFromExpr(mdata design.MetadataExpr) map[string]interface{} {
```

Comment:
why does this have to be a public method?

Reply:
It is used in the security plugin

Reply:
https://github.com/goadesign/plugins/blob/security_middleware/security/generate.go#L163

Reply:
got it, thanks!